.github/workflows/scripts/slack_gem_notifications/notifications_methods.rb

```diff
@@ -0,0 +1,48 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require 'time'
+require 'httparty'
+
+def check_for_updates(watched_gems)
+  return if gem_list_empty?(watched_gems)
+  watched_gems.each do |gem_name|
+    is_valid_gem?(gem_name) ? gem_info = is_valid_gem?(gem_name) : return
```

Comment:
When a method ends in a question mark, I expect it to return a boolean. With the refactored scope of `is_valid_gem?`, perhaps a different name is in order?